HTML/mockup-corporate/js/task4.js

```diff
@@ -0,0 +1,15 @@
+import {elementsLoaded} from './elements.js';
+
+export const playPauseButton = () => {
```

Comment:
maybe something with video would be more descriptive

Reply:
like `initVideoControls`